npm/webpack-dev-server/src/measureWebpackPerformance.ts

```diff
@@ -13,7 +13,7 @@ export function measureWebpackPerformance (webpackConfig: webpack.Configuration)
   const compareWithPrevious = process.env.WEBPACK_PERF_MEASURE_COMPARE
 
   function percentageDiff (a: number, b: number) {
```

Comment:
Maybe using the word `before` and `after` would make this function easier to use.

Reply:
It's not actually before and after but any number. Maybe base and comparing?